src/Select/SelectInput.js

```diff
@@ -114,15 +114,21 @@ class SelectInput extends React.Component<AllProps, State> {
     });
   };
 
-  handleItemClick = (child: Element<*>) => (event: SyntheticMouseEvent<>) => {
+  handleItemClick = (child: Element<*>) => (event: SyntheticMouseEvent<> & { target?: Object }) => {
```

Comment:
Event is not in our control, therefore we should not be fabricating a type.  _It is what it is_, so you must add a dynamic check (type refinement) to see if target is present.

Reply:
There already is a type refinement for `target`, so the last thing to do is to change `target`'s type to `any`?